backend/app/Action/Auth/UploadProfileImageAction.php

```diff
@@ -33,7 +33,9 @@ public function execute(UploadProfileImageRequest $request): UploadProfileImageR
             $user->id,
             'public'
         );
-        $user->profile_image = $disk->url($filePath);
+
+        // added timestamp to url to let browser know that image was changed
+        $user->profile_image = $disk->url($filePath) . '?t=' . time();
```

Comment:
Maybe no need for timestamp because url will be changed everytime

Reply:
@nemoipaha in this case url always be the same, because we are using user id in the file path

Reply:
Probably this is a correct way to store file path
```
$filePath = Storage::putFileAs(
            Config::get('filesystems.profile_images_dir'),
            $request->getImage(),
            $request->getImage()->hashName(),
            'public'
);